pom.xml

```diff
@@ -276,16 +276,18 @@
     </build>
 
     <modules>
-        <module>sdk-server-common</module>
+        <module>client</module>
         <module>common</module>
-        <module>grpc</module>
+        <module>examples/helloworld</module>
+        <module>reference/grpc</module>
+        <module>reference/jsonrpc</module>
+        <module>sdk-server-common</module>
```

Comment:
Would it make sense to rename `sdk-server-common` to just `server-common` to match the format being used for the other modules?